ImageFeed/ImagesList/ImagesListCell.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+final class ImagesListCell: UITableViewCell {
```

Comment:
Давай договоримся, между разными объявлениями оставлять одну пустую строку. Между функциями, между классом и списком импорта

```suggestion
import UIKit

final class ImagesListCell: UITableViewCell {
```

Reply:
Принял к сведению. в проекте поправил.